osu.Game/Overlays/Rankings/RankingsSortTabControl.cs

```diff
@@ -1,19 +1,42 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System;
+using osu.Framework.Localisation;
+using osu.Game.Resources.Localisation.Web;
+
 namespace osu.Game.Overlays.Rankings
 {
     public class RankingsSortTabControl : OverlaySortTabControl<RankingsSortCriteria>
     {
         public RankingsSortTabControl()
         {
-            Title = "Show";
+            Title = RankingsStrings.FilterTitle;
```

Comment:
Should be uppercased.

![2021-07-30-190241_482x360_scrot](https://user-images.githubusercontent.com/20418176/127687447-d08a030a-dc8e-48ee-a141-dfbc1e594001.png)

Layout is also different web-side but that's out of scope for now.